src/main/java/edu/hm/hafner/analysis/parser/MsBuildParser.java

```diff
@@ -25,7 +25,7 @@ public class MsBuildParser extends LookaheadParser {
             + ANT_TASK + "(?:(?:\\s*(?:\\d+|\\d+:\\d+)>)?(?:(?:(?:(?<a4>.*?)\\((?<a5>\\d*)(?:,(?<a6>\\d+))?[a-zA-Z]*?\\)|.*LINK)\\s*:|"
             + "(?<a7>.*):)\\s*(?<a8>[A-z-_]*\\s(?:[Nn]ote|[Ii]nfo|[Ww]arning|(?:fatal\\s*)?[Ee]rror))[^A-Za-z0-9]\\s*:?\\s*(?<a9>[A-Za-z0-9\\-_]+)?"
             + "\\s*:\\s(?:\\s*(?<a10>[A-Za-z0-9.]+)\\s*:)?\\s*(?<a11>.*?)(?: \\[(?<a12>[^\\]]*)[/\\\\][^\\]\\\\]+\\])?"
-            + "|(?<a13>.*)\\s*:.*(?<a14>:error|warning)\\s*(?<a15>LNK[0-9]+):\\s*(?<a16>.*)))$";
+            + "|(?<a13>.*)\\s*:.*(?<a14>error|warning)\\s*(?<a15>LNK[0-9]+):\\s*(?<a16>.*)))$";
```

Comment:
Did you mean removing the group name from here @uhafner ?

Reply:
In `createIssue `numbers are used.
```

 if (StringUtils.isNotBlank(matcher.group(2))) {
            return builder.setLineStart(0)
                    .setCategory(matcher.group(1))
                    .setMessage(matcher.group(2))
                    .setSeverity(Severity.WARNING_NORMAL)
                    .buildOptional();
        }
```

When I try to use numbers I get:

```
java.util.regex.PatternSyntaxException: capturing group name does not start with a Latin letter near index 34
```

Reply:
No, see https://github.com/jenkinsci/analysis-model/blob/f805dea38333c64cf0caaf8d2e71cc2195c6cf83/src/main/java/edu/hm/hafner/analysis/parser/EmbeddedEngineerParser.java#L77

The `:` is the separator between two entries. It is required but not as part of the word error or warning. 

The group should be named `<severity>`